bench_groupby.py

```diff
@@ -18,3 +18,25 @@
 def test_groupby(benchmark, ncols, nkey_cols, nrows):
     frame = utils.make_frame(ncols, nkey_cols, nrows)
     benchmark(frame.groupby, by=[f"key{i}" for i in range(nkey_cols)])
+
+
+@pytest.mark.parametrize(
+    "nrows",
+    [100, 10_000, 1_000_000],
+)
+@pytest.mark.parametrize(
+    "agg",
+    ["sum", ["sum", "mean"], {f"val{i}": ["sum", "mean", "count"] for i in range(6)}],
```

Comment:
It's a little brittle that we hardcode 6 here since that will break if we reduce the number of columns below, but I think it's fine for now.